Extension/package.json

```diff
@@ -3376,6 +3376,12 @@
                         "default": "default",
                         "markdownDescription": "%c_cpp.configuration.copilotHover.markdownDescription%",
                         "scope": "window"
+                    },
+                    "C_Cpp.persistVsDeveloperEnvironment": {
```

Comment:
I have this set to false, but it's still persisting, i.e. the setting doesn't seem to be working, even when I reload window.

Reply:
Oh, maybe I'm misunderstanding what the setting is supposed to do? When I reload window, is the vs developer environment supposed to be cleared on startup the next time?

Reply:
If you have it set to false, our environment override should not be there the next time you start a session in this workspace. Are you still running in the Developer Command Prompt and maybe confusing that environment with the environment override from the Set VS Developer Environment command?

And the value of this setting is applied at the time you run the command. So maybe the feedback here is that we should notify the user that the setting will not take effect until the next time you run the command.  Or maybe there's a way for me to reset the override to use the new setting value.

Reply:
I'll see if I can change the persist value on the fly when the user changes it.

Reply:
Ok, the setting is true and I run the Set and then change the setting to false, it still persists, but it doesn't persist if I use the Set when the setting is true -- is that intentional? I would've expected previously called Set's to not persist when the setting changed to false without having to manually run the Clear.

Reply:
Changing it as a reaction to a settings change seems to work fine. So you will get whatever behavior the setting is currently set to when you close the window.